src/transformers/models/electra/modeling_electra.py

```diff
@@ -691,6 +692,15 @@ def _set_gradient_checkpointing(self, module, value=False):
         if isinstance(module, ElectraEncoder):
             module.gradient_checkpointing = value
 
+    def update_keys_to_ignore(self, config, del_keys_to_ignore):
```

Comment:
is this necessary? is it ever called actually?

Reply:
Actually looks like it's not necessary. I'll remove it then.